tools/doc/html.js

```diff
@@ -288,7 +288,7 @@ var BSD_ONLY_SYSCALLS = new Set(['lchmod']);
 // Returns modified text, with such refs replace with HTML links, for example
 // '<a href="http://man7.org/linux/man-pages/man2/open.2.html">open(2)</a>'
 function linkManPages(text) {
-  return text.replace(/ ([a-z.]+)\((\d)\)/gm, function(match, name, number) {
+  return text.replace(/ ([a-z.]+)\((\d[a-z]?)\)/gm, function(match, name, number) {
```

Comment:
we need both the original and new match for this to return http://man7.org/linux/man-pages/man1/fg.1p.html

currently it will return `/man-pages/man1p/` by the looks of it.